base/allocator/partition_allocator/partition_alloc.gni

```diff
@@ -47,7 +47,9 @@ declare_args() {
   #     and doesn't wish to incur the library size increase (crbug.com/674570).
   # 2.  On NaCl (through this declaration), where PartitionAlloc doesn't
   #     build at all.
-  use_partition_alloc = !is_nacl
+
+# TODO(cobalt, b/382115610): Re-enable use_partition_alloc
```

Comment:
This should probably be done in args.gn files

Reply:
duh, i got a bit lost looking at all the base changes and didnt realize this :) . thanks for pointing this out